api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/context/ReportPaymentMessageContext.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.dhsc.htbhf.claimant.message.context;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.reporting.PaymentAction;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Value
+@Builder
+public class ReportPaymentMessageContext {
+    private Claim claim;
+    private Optional<PaymentCycle> paymentCycle;
```

Comment:
Why is the PaymentCycle Optional?

Reply:
Because for additional pregnancy payments we _might_ not have a PaymentCycle